<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportTable</class>
 <widget class="QDialog" name="ImportTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="rowTab">
    <attribute name="title">
     <string>Lipid row (pivot) table</string>
    </attribute>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Sample columns*</string>
     </property>
    </widget>
    <widget class="QListWidget" name="sampleListWidgetRow">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>221</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="okButtonRow">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>410</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButtonRow">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>410</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QListWidget" name="ignoreListWidgetRow">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>221</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Lipid column*</string>
     </property>
    </widget>
    <widget class="SingleListWidget" name="lipidListWidgetRow">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Input columns</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="columnTab">
    <attribute name="title">
     <string>Lipid column (pivot) table</string>
    </attribute>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>70</y>
       <width>191</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Numerical study varibles</string>
     </property>
    </widget>
    <widget class="SingleListWidget" name="sampleListWidgetCol">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Input columns</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButtonCol">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>410</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="okButtonCol">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>410</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Sample column*</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lipidListWidgetCol">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>221</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="ignoreListWidgetCol">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>221</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Lipid columns*</string>
     </property>
    </widget>
    <widget class="QListWidget" name="numericalFeatureListWidgetCol">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>90</y>
       <width>221</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>240</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Nominal study variables</string>
     </property>
    </widget>
    <widget class="QListWidget" name="nominalFeatureListWidgetCol">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>260</y>
       <width>221</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMappingCol">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>410</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Mapping of study variables</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="flatTab">
    <attribute name="title">
     <string>Flat table</string>
    </attribute>
    <widget class="SingleListWidget" name="lipidListWidgetFlat">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="ignoreListWidgetFlat">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>221</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>181</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Numerical study variables</string>
     </property>
    </widget>
    <widget class="SingleListWidget" name="quantListWidgetFlat">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>200</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>181</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Quant data column*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Input columns</string>
     </property>
    </widget>
    <widget class="QPushButton" name="okButtonFlat">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>410</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QListWidget" name="numericalFeatureListWidgetFlat">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButtonFlat">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>410</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Sample column*</string>
     </property>
    </widget>
    <widget class="SingleListWidget" name="sampleListWidgetFlat">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Lipid column*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>210</y>
       <width>201</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Nominal study variables</string>
     </property>
    </widget>
    <widget class="QListWidget" name="nominalFeatureListWidgetFlat">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>230</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMappingFlat">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>410</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Mapping of study variables</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="previewBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>751</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Preview of the loaded table</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>731</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SingleListWidget</class>
   <extends>QListWidget</extends>
   <header>lipidspace/AssistanceFunctions.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
